generatedTypes/lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView/KeyboardTrackingView.ios.d.ts

```diff
@@ -13,7 +13,6 @@ import { KeyboardTrackingViewProps } from './index';
  */
 declare class KeyboardTrackingView extends PureComponent<KeyboardTrackingViewProps> {
     static displayName: string;
-    /** V6 should be change to default false */
```

Comment:
This doesn't seem relevant to this PR? 
Did you already change the useSafeArea default on another PR?

Reply:
Yes. It's already merged